apps/etterlatte-fordeler/src/main/kotlin/fordeler/FordelerService.kt

```diff
@@ -13,39 +13,75 @@ import java.time.Clock
 import java.time.OffsetDateTime
 import java.time.format.DateTimeFormatter
 
+enum class Vedtaksloesning {
+    DOFFEN, PESYS
+}
+data class Fordeling(val soeknadId: Long, val fordeltTil: Vedtaksloesning, val kriterier: List<FordelerKriterie>)
+
 sealed class FordelerResultat {
     object GyldigForBehandling : FordelerResultat()
     class UgyldigHendelse(val message: String) : FordelerResultat()
     class IkkeGyldigForBehandling(val ikkeOppfylteKriterier: List<FordelerKriterie>) : FordelerResultat()
 }
 
+fun featureFromEnv(featureName: String): Long {
+    return (System.getenv()[featureName]?.toLong() ?: 0).takeIf { it > -1 } ?: Long.MAX_VALUE
+}
+
 class FordelerService(
     private val fordelerKriterier: FordelerKriterier,
     private val pdlTjenesterKlient: PdlTjenesterKlient,
-    private val klokke: Clock = Clock.systemUTC()
+    private val fordelerRepository: FordelerRepository,
+    private val klokke: Clock = Clock.systemUTC(),
+    private val maxFordelingTilDoffen: Long = featureFromEnv("FEATURE_MAX_FORDELING_TIL_DOFFEN")
```

Comment:
Jeg liker jo denne typen for featuretoggles men siden vi har et forslag på unleash burde vi kanskje samskjøre dette?
ref https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/738/files

Reply:
Tenkte at siden denne appen skal bort på sikt, så kunne jeg gjøre det enkelt. Ble vi enige om at Unleash skulle brukes uansett hvor enkelt behovet for feature-toggling er?

Reply:
Trur vi gjer livet lettare for oss sjølv med å ha alt samme plass. Så når vi startar å bruke Unleash bør vi bruke det for all toggling.

Reply:
Samtidig så trur eg ikkje Unleash har støtte for så-og-så-mange-type-toggles ut av boksen, så for å ta deti bruk her må vi lage noko lurium sjølv. Kanskje ikkje nyttig.